generators/client/templates/angular/src/main/webapp/app/shared/login/_login.component.html

```diff
@@ -23,7 +23,7 @@ <h4 class="modal-title" jhiTranslate="login.title">Sign in</h4>
 </div>
 <div class="modal-body">
     <div class="row">
-        <div class="col-md-4 offset-md-4">
+        <div class="col-md-4 offset-md-2">
```

Comment:
bootstrap grid size is 12, but 4+2*2=8 <> 12  
maybe better is `class="col-md-6 offset-md-3"`